x/emissions/proto/emissions/state/v1/tx.proto

```diff
@@ -27,74 +28,100 @@ service Msg {
+  rpc RemoveFromTopicCreationWhitelist(MsgRemoveFromTopicCreationWhitelist)
+      returns (MsgRemoveFromTopicCreationWhitelistResponse);
 
-  rpc AddToWeightSettingWhitelist(MsgAddToWeightSettingWhitelist) returns (MsgAddToWeightSettingWhitelistResponse);
+  rpc AddToWeightSettingWhitelist(MsgAddToWeightSettingWhitelist)
+      returns (MsgAddToWeightSettingWhitelistResponse);
 
-  rpc RemoveFromWeightSettingWhitelist(MsgRemoveFromWeightSettingWhitelist) returns (MsgRemoveFromWeightSettingWhitelistResponse);
+  rpc RemoveFromWeightSettingWhitelist(MsgRemoveFromWeightSettingWhitelist)
+      returns (MsgRemoveFromWeightSettingWhitelistResponse);
 }
 
-
 // Because gocosmos, grpc-gateway, and go-pulsar do not support optional fields
-// and including google themselves https://cloud.google.com/apis/design/design_patterns.md#optional_primitive_fields
-// we instead use a repeated field with a single element to represent an optional field
-// and if the repeated field is empty, it is considered to be the same as if the field was not set
+// and including google themselves
+// https://cloud.google.com/apis/design/design_patterns.md#optional_primitive_fields
+// we instead use a repeated field with a single element to represent an
+// optional field and if the repeated field is empty, it is considered to be the
+// same as if the field was not set
 message OptionalParams {
-  repeated string version = 1; // version of the protocol should be in lockstep with github release tag version
-  repeated int64 epoch_length = 2; // length of an "epoch" for rewards payouts in blocks
+  repeated string version = 1; // version of the protocol should be in lockstep
+                               // with github release tag version
+  repeated int64 epoch_length =
+      2; // length of an "epoch" for rewards payouts in blocks
   repeated string emissions_per_epoch = 3 [
-    (cosmos_proto.scalar)  = "cosmos.Int",
+    (cosmos_proto.scalar) = "cosmos.Int",
     (gogoproto.customtype) = "cosmossdk.io/math.Int",
-    (gogoproto.nullable)   = false,
+    (gogoproto.nullable) = false,
     (amino.dont_omitempty) = true
   ]; // default amount of tokens to issue per epoch
   repeated string min_topic_unmet_demand = 4 [
-    (cosmos_proto.scalar)  = "cosmos.Uint",
+    (cosmos_proto.scalar) = "cosmos.Uint",
     (gogoproto.customtype) = "cosmossdk.io/math.Uint",
-    (gogoproto.nullable)   = false,
+    (gogoproto.nullable) = false,
     (amino.dont_omitempty) = true
-  ]; // total unmet demand for a topic < this => don't run inference solicatation or weight-adjustment
-  repeated uint64 max_topics_per_block = 5; // max number of topics to run cadence for per block
+  ]; // total unmet demand for a topic < this => don't run inference
+     // solicatation or weight-adjustment
+  repeated uint64 max_topics_per_block =
+      5; // max number of topics to run cadence for per block
   repeated string min_request_unmet_demand = 7 [
```

Comment:
No field number 6 here. Is that jump done for a reason? It wont affect Proto compiler but it just looks better.

Reply:
true

Reply:
fixed in #56